.github/free-disk-space.sh

```diff
@@ -22,7 +22,7 @@
 # Free: 17 GB
 # This script frees up 28 GB of disk space by deleting unneeded packages and 
 # large directories.
-# The Flink end to end tests download and generate more than 17 GB of files,
+# The following end to end tests download and generate more than 17 GB of files,
```

Comment:
It should be `Flink` as the script is referred from the flink's repo.

Reply:
Do you want me to rebase and drop the commit or revert? (or something else?)

Reply:
You could revert the commit that touched this line.